src/main/target/common_post.h

```diff
@@ -329,6 +329,12 @@
 #endif
 #endif
 
+#if defined(USE_FLASH_W25N02K) && !defined(USE_FLASH_W25N01G)
+#if !defined(USE_FLASH_W25N01G)
+#define USE_FLASH_W25N01G
+#endif
+#endif
```

Comment:
```suggestion
#if defined(USE_FLASH_W25N02K) && !defined(USE_FLASH_W25N01G)
#define USE_FLASH_W25N01G
#endif
```

Reply:
There were other identical constructs so I'd just copied this. I've now fixed them all.